### PR TITLE
AOM-72: Add icon to indicate that add-on is running

### PR DESCRIPTION
**Pull request by malmike**
## JIRA TICKET NAME:
[AOM-72 Add icon to indicate that add-on is running](https://issues.openmrs.org/browse/AOM-72)
### SUMMARY:
Add icons on the index page of add-on manager to indicate which add-ons are running and which one are not

### CODE REVIEW

**Comment by mkiterian** [comment]
Looks good to me

**Comment by Annettesunday** [comment]
Looks great!

File: app/js/components/manageApps/AddonList.jsx [hunk 1]
```diff
@@ -30,7 +30,15 @@ export const AddonList = ({ handleDownload, appList, openPage, openModal, update
                 />
               </td>
               <td onClick={() => openPage(app.appDetails)}>
-                <div className="addon-name">{app.appDetails && app.appDetails.name}</div>
+                <div className="addon-name">
+                  {
+                    app.appDetails.started == true || app.appDetails.started == undefined ?
```
Comment (tduyile04) [inline app/js/components/manageApps/AddonList.jsx]: Using double equal too sign is not a sure way to check equality considering javascript's implicit coercion. Using a `===` is better. Also you can re-write this line with `{ app.appDetails && app.appDetails.started  ? pass condition : fail condition }`

File: tests/components/AddonList.test.js [hunk 2]
```diff
@@ -40,21 +40,21 @@ describe('<AddonList />', () => {
     const openPage = () => {};
     const openModal = () => {};
     const addonList = mount( <AddonList appList={appList} openPage={openPage}  openModal={openModal}/ > );
-
```
Comment (LWanjiru) [inline tests/components/AddonList.test.js:43]: Consider : I observe that this PR contains a lot of format and spacing changes that are not quite relevant to  the task. Daniel recommended that it is wise to separate formatting changes from the changes that show what directly applies to  the feature you were working on. This way,  the PR becomes less noisy.

File: app/js/components/manageApps/AddonList.jsx [hunk 3]
```diff
@@ -30,7 +30,15 @@ export const AddonList = ({ handleDownload, appList, openPage, openModal, update
                 />
               </td>
               <td onClick={() => openPage(app.appDetails)}>
-                <div className="addon-name">{app.appDetails && app.appDetails.name}</div>
+                <div className="addon-name">
+                  {
+                    app.appDetails.started === true || app.appDetails.started === undefined ?
```
Comment (LWanjiru) [inline app/js/components/manageApps/AddonList.jsx:35]: Why does it return `started status` when `app.appDetails.started` evaluates to undefined? What if the contents are undefined owing to a bug and the task has not been started? What happens then?

Comment (malmike) [reply app/js/components/manageApps/AddonList.jsx:35]: The owa's do not have a started property so when you call app.appDetails.name it returns undefined. The started condition mainly applies to a module as for owa's they are alway running

File: tests/components/AddonList.test.js (hunk 2 above)
Comment (malmike) [reply tests/components/AddonList.test.js:43]: My editor's linter automatically removes some linting issues like white space

**Pull request merged**